deb/gen-deb-ver

```diff
@@ -13,39 +13,20 @@ GIT_COMMAND="git -C $REPO_DIR"
 origVersion="$VERSION"
 debVersion="${VERSION#v}"
 
-gen_deb_version() {
-	# Adds an increment to the deb version to get proper order
-	# 18.01.0-tp1   -> 18.01.0-0.1-tp1
-	# 18.01.0-beta1 -> 18.01.0-1.1-beta1
-	# 18.01.0-rc1   -> 18.01.0-2.1-rc1
-	# 18.01.0       -> 18.01.0-3
-	fullVersion="$1"
-	pattern="$2"
-	increment="$3"
-	testVersion="${fullVersion#*-$pattern}"
-	baseVersion="${fullVersion%-"$pattern"*}"
-	echo "$baseVersion-$increment.${testVersion##.}.$pattern$testVersion"
-}
-
-case "$debVersion" in
-	*-dev)
-		;;
-	*-tp[.0-9]*)
-		debVersion="$(gen_deb_version "$debVersion" tp 0)"
-		;;
-	*-beta[.0-9]*)
-		debVersion="$(gen_deb_version "$debVersion" beta 1)"
-		;;
-	*-rc[.0-9]*)
-		debVersion="$(gen_deb_version "$debVersion" rc 2)"
-		;;
-	*)
-		debVersion="$debVersion-3"
```

Comment:
Need to make some changes to preserve this `~3` for the 20.10 branch (for now) so that the `docker-ce`, `docker-ce-cli`, and `docker-ce-rootless-extras` packages keep it (23.0.0 will start using the new versioning scheme)